core_tic_tac_toe/medium_computer_player.py

```diff
@@ -0,0 +1,68 @@
+class MediumComputerPlayer:
+
+    def __init__(self, name, symbol, board, opponent_symbol):
+        self.__name = name
+        self.__symbol = symbol
+        self.__board = board
+        self.__opponent_symbol = opponent_symbol
+
+    @property
+    def symbol(self):
+        return self.__symbol
+
+    @property
+    def board(self):
+        return self.__board
+
+    @property
+    def name(self):
+        return self.__name
+
+    @property
+    def game_logic(self):
+        return self.__game_logic
+
+    @property
+    def opponent_symbol(self):
+        return self.__opponent_symbol
+
+    def make_move(self):
+        horizontals_verticals_diagonals = self.board.get_all_positions()
+        available_positions = self.board.get_available_positions()
+
+        move_to_win = self.check_for_win(
+            horizontals_verticals_diagonals, self.symbol, available_positions)
+        move_to_block = self.check_for_win(
+            horizontals_verticals_diagonals, self.opponent_symbol, available_positions)
+
+        computer_move = self.__get_computer_move(
+            move_to_win, move_to_block, available_positions)
+
+        self.board.execute_move(computer_move - 1, self.symbol)
```

Comment:
Why subtract 1?

If you calculate that there's a move to win, why also calculate a move to block?